### DISCUSSION TITLE
FulfillmentInbound: Error converting value 0.14 to type 'FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.BigDecimalType

### DISCUSSION THREAD

**Discussion by IncredibleX** [abuzuhri/Amazon-SP-API-CSharp#148] [Q&A] [answered]
Hi,

when i create a shipmentplan i got the folliwing message.
Error converting value 0.14 to type 'FikaAmazonAPI.AmazonSpApiSDK.Models.FulfillmentInbound.BigDecimalType'. Path 'payload.InboundShipmentPlans[0].EstimatedBoxContentsFee.FeePerUnit.Value', line 53, position 62.

Any Idea?

Thanx,

Tobias

**Comment by abuzuhri** [comment]
BigDecimalType not used any  more which version you are using ?

**Comment by IncredibleX** [comment, marked as answer]
Thanx - i found out, that i was using an old version. Now it works.

Tobias Stephan
eCommerce & Softwaredevelopment
+49 171 3833568
***@***.***
Shops:
www.leckerstecker.de
www.herr-der-dinge.de

Softwareblog:
www.exoda.de
Kookamp 40, 46354 Südlohn
Apps:
Leckerstecker Kabelshop
3D Print for Dremel 3D45 3D40
Listory Lister Shopping App
MCR-Tools

...noch mehr

Am 24. März 2022, 16:24 +0100 schrieb Tareq Abuzuhri ***@***.***>:

**Comment by abuzuhri** [comment]
Please check all others points and let me know if you need any help
